keyboards/7liang/tp32/config.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2022 Pkerr (@sky_jun@qq.com)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID    0x1503

```

Comment:
This vendor ID is already in use. 
https://www.the-sz.com/products/usbid/index.php?v=0x1503&p=&n=

Reply:
I learned, thanks